yoti_python_sdk/doc_scan/session/retrieve/face_capture_response.py

```diff
@@ -0,0 +1,28 @@
+from .media_response import MediaResponse
+from .resource_response import ResourceResponse
+
+
+class FaceCaptureResponse(ResourceResponse):
+    """
+    Represents a Face Capture resource for a given session
+    """
+
+    def __init__(self, data=None):
+        self._data = data or {}
+        ResourceResponse.__init__(self, data)
+
+    @property
+    def id(self):
+        return self._data.get("id")
+
+    @property
+    def source(self):
+        return self._data.get("source")
+
+    @property
+    def image(self):
+        return MediaResponse(data=self._data.get("image") or {})
+
+    @property
+    def tasks(self):
+        return self._data.get("tasks") or []
```

Comment:
Does this line do anything, correct me if ii'm wrong but won't this always return `self._data.get("tasks")` ?

```suggestion
        return self._data.get("tasks") if len(self._data.get("tasks")) > 0 else []
```

Reply:
in python return self._data.get("tasks") or [] will work same like Your solution:

self._data.get("tasks") -> if its [] or None it will use shortcut value -> []
return self._data.get("tasks") if len(self._data.get("tasks")) > 0 else []

in python one liners are consinder as anti pattern. In 90 % cases You can use shortcuts that are more readable and user friendly.

Summary - current state of the code works the same like Your proposition

Reply:
Cool thanks for clarifying.